src/app/modals/about-modal.component.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  Component,
+  DoCheck,
+  Input,
+  Output,
+  EventEmitter,
+  TemplateRef,
+  ViewEncapsulation,
+  OnInit
+} from '@angular/core';
+
+import { cloneDeep, defaults, isEqual } from 'lodash';
+
+import { AboutModalConfig } from './about-modal-config';
+
+/**
+ * Component for rendering AboutModal
+ */
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'pfng-about-modal',
+  templateUrl: './about-modal.component.html'
+})
+
+export class AboutModalComponent implements DoCheck, OnInit {
+
+  /**
+   * The AboutModal config contaning component properties
+   */
+  @Input() config: AboutModalConfig;
+
+  /**
+   * Modal Template for custom content
+   */
+  @Input() template: TemplateRef<any>;
+
+  /**
+   * The Event is emitted when modal is closed
+   */
+  @Output('onCancel') onCancel = new EventEmitter();
+
+  private defaultConfig= {
+    isOpen: false
```

Comment:
You can remove isOpen, but can leave the defaultConfig. We may want to add to this later?

Add a space before the equals for defaultConfig=